### PR TITLE
usb: logs: Use original logger functions

### PR DESCRIPTION
**Pull request by finikorg**
When CONFIG_LOG_FUNCTION_NAME is defined to not add name again.

Signed-off-by: Andrei Emeltchenko <andrei.emeltchenko@intel.com>

### CODE REVIEW

**Review by aurel32 — APPROVED** [review]
This looks a fine to me, and indeed fixes the double function name in the logs. Given  `LOG_FUNCTION_NAME` is enabled by default, I wonder if we can even go further by always dropping the function name.

File: include/usb/usb_device.h [hunk 1]
```diff
@@ -44,8 +44,13 @@
 extern "C" {
 #endif
 
+#if defined(CONFIG_LOG_FUNCTION_NAME)
```
Comment (nashif) [inline include/usb/usb_device.h]: now that logging function name is supported by the logging subsystem, shouldn't we remove all of this and move from USB_ to LOG_ directly?

**Review by nashif — CHANGES_REQUESTED** [review]

File: include/usb/usb_device.h (hunk 1 above)
Comment (finikorg) [reply include/usb/usb_device.h]: Yes, I will do this.

**Comment by pfalcon** [comment]
> usb: logs: Use original logger functions

This sounds confusing to me. Do you switch from new CONFIG_LOG to the original CONFIG_SYS_LOG?

File: include/usb/usb_device.h (hunk 1 above)
Comment (loicpoulain) [reply include/usb/usb_device.h]: yes please remove

**Comment by finikorg** [comment]
> > usb: logs: Use original logger functions
> 
> This sounds confusing to me. Do you switch from new CONFIG_LOG to the original CONFIG_SYS_LOG?

No

File: include/usb/usb_device.h (hunk 1 above)
Comment (finikorg) [reply include/usb/usb_device.h]: BTW, this is done.

**Comment by nashif** [comment]
> I was trying to make every commit buildable and logically complete.

the whole thing could have been done in  1 single commit :-)

**Review by nashif — APPROVED** [review]

**Pull request merged**